# Campaign Scenario Analysis

### 1. Business Question

This analysis estimates the potential incremental sales from targeting 10 high-value, frequent households identified as untapped promotional opportunities. These households demonstrate strong purchasing behaviour but have no recorded campaign redemption history. The objective is to size a plausible revenue opportunity under conservative and optimistic response scenarios, using the observed spending difference between exposed and non-exposed households within comparable value tiers as the evidence base.

### 2. Target Households

The target group consists of the 10 highest-scoring households among those with no recorded campaign history. All 10 households belong to the **High Value - Frequent** segment, indicating strong historical purchasing activity alongside an absence of recorded promotional exposure.

Their existing sales are used as the baseline against which the campaign scenarios will be evaluated.

In [1]:
import pandas as pd

target_households = pd.DataFrame({
    "household_key": [
        2459, 1023, 900, 1489, 707,
        909, 1510, 328, 591, 2312
    ],
    "baseline_sales": [
        20671.50, 38319.79, 16450.53, 17251.53, 19194.42,
        12414.56, 11211.37, 17332.13, 13534.93, 14682.81
    ],
    "value_segment": [
        "High Value - Frequent"
    ] * 10
})

target_households

,household_key,baseline_sales,value_segment
0,2459,20671.50,High Value - Frequent
1,1023,38319.79,High Value - Frequent
2,900,16450.53,High Value - Frequent
3,1489,17251.53,High Value - Frequent
4,707,19194.42,High Value - Frequent
5,909,12414.56,High Value - Frequent
6,1510,11211.37,High Value - Frequent
7,328,17332.13,High Value - Frequent
8,591,13534.93,High Value - Frequent
9,2312,14682.81,High Value - Frequent


In [2]:
baseline_sales = target_households["baseline_sales"].sum()

print(f"Target households: {len(target_households)}")
print(f"Baseline sales: ${baseline_sales:,.2f}")

Target households: 10
Baseline sales: $181,063.57


### 3. Scenario Assumptions

The 10 untapped households are all classified as Value Tier 4. In the tier-stratified comparison, Value Tier 4 households showed lower average sales among exposed households than non-exposed households. Therefore, the analysis does not assume a positive campaign effect based on the observed data.

The current scenario is treated as a sensitivity analysis:
- **Current / Evidence-based:** 0% uplift
- **Illustrative upside:** 10% uplift

The 10% upside is not an estimated campaign effect; it is an illustrative sensitivity case used to show the potential revenue scale if a future campaign generated a modest positive response.

In [3]:
scenario_results = pd.DataFrame({
    "scenario": ["Current", "Illustrative Upside"],
    "uplift_pct": [0.00, 0.10]
})

scenario_results["estimated_sales"] = (
    baseline_sales * (1 + scenario_results["uplift_pct"])
)

scenario_results["incremental_sales"] = (
    scenario_results["estimated_sales"] - baseline_sales
)

scenario_results["uplift_pct"] = (
    scenario_results["uplift_pct"] * 100
)

scenario_results.round(2)

,scenario,uplift_pct,estimated_sales,incremental_sales
0,Current,0.0,181063.57,0.00
1,Illustrative Upside,10.0,199169.93,18106.36


### 4. Scenario Results

The 10 selected households currently represent approximately **$181.1K in historical sales**. Under the illustrative 10% upside scenario, this would correspond to approximately **$199.2K in sales**, or **$18.1K in potential incremental sales**.

This is a sensitivity case rather than a campaign forecast. The uplift is illustrative because the observed Tier 4 comparison does not provide evidence of a positive campaign effect.

### 5. Business Interpretation

The 10 identified untapped households currently represent **$181,063.57** in historical sales. Because all ten households fall into Value Tier 4, the Stage 4 comparison does not provide evidence of a positive campaign uplift for this specific segment; exposed Tier 4 households actually recorded lower average sales than non-exposed households. Therefore, the analysis does not claim that targeting these households will generate incremental revenue. As a sensitivity case, a hypothetical 10% uplift would increase sales to approximately **$199,169.93**, representing **$18,106.36** in potential incremental sales, but this should be treated as illustrative rather than a forecast.

In [4]:
# Power BI export: campaign scenario

campaign_scenario = scenario_results.copy()

campaign_scenario.to_csv(
    "campaign_scenario.csv",
    index=False
)

campaign_scenario

,scenario,uplift_pct,estimated_sales,incremental_sales
0,Current,0.0,181063.570,0.000
1,Illustrative Upside,10.0,199169.927,18106.357
